examples/express-composition/src/__tests__/acceptance/test-helper.ts

```diff
@@ -0,0 +1,60 @@
+import {givenHttpServerConfig, Client, supertest} from '@loopback/testlab';
+import {Request, Response} from 'express';
+import {NoteApplication} from '../../note-application';
+import {ExpressApplication} from '../../application';
+import {Note} from '../../models/note.model';
+import * as path from 'path';
+import * as http from 'http';
+import * as pEvent from 'p-event';
+
+export async function setupExpressApplication(): Promise<AppWithClient> {
```

Comment:
👎 

IIUC, you are duplicating the code setting up the Express application here in tests again.